internal/server/data/data.go

```diff
@@ -35,7 +35,16 @@ func NewDB(connection gorm.Dialector) (*gorm.DB, error) {
 	return db, nil
 }
 
-func newRawDB(connection gorm.Dialector) (*gorm.DB, error) {
+func PreMigrate(db *gorm.DB) error {
+	return premigrate(db)
+}
+
+func Migrate(db *gorm.DB) error {
+	return migrate(db)
+}
```

Comment:
We can remove these and export `migrate` and `premigrate` since they are the same function

```suggestion
```